src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.scss

```diff
@@ -3,9 +3,11 @@
 @import '@gravity-ui/uikit/styles/mixins.scss';
 
 .healthcheck {
-    // Since most of the inner containers have fixed width, we can set fixed width here as well
-    // Thus we will get rid of unneeded layout shift when scrollbar appear
-    min-width: 885px;
+    &_expanded {
+        // Since most of the inner containers have fixed width, we can set fixed width here as well
+        // Thus we will get rid of unneeded layout shift when scrollbar appear
+        min-width: 885px;
+    }
```

Comment:
`min-width` should apply only to the healthcheck in modal, not in the preview mode